src/wallet/wallet.ts

```diff
@@ -94,6 +95,7 @@ class HathorWalletServiceWallet extends EventEmitter implements IHathorWallet {
   private xpub: string | null;
   // Xpriv of the wallet on the account derivation path
   private xpriv: string | null;
+  private authxpriv: string | null;
```

Comment:
Missing comment, which path is this xpriv from?

Reply:
Removed from the properties, but it is still on the constructor arguments.

The reson for this is that we used to expect the auth xpriv to be persisted through the access data on storage, but if the access data is persisted, we do not need to use the xpriv to derive the access data since the persisted storage will have the entire access data.

Since we want to preserve this feature we need to have the auth xpriv either derived or passed as an argument.
And this facade expects the xpriv to be the account path xpriv which cannot be used to derive the auth path xpriv.
So the only option is to require the auth key xpriv in the constructor.

I removed the `authxpriv` property since as you pointed out, we already have a property for this.